packages/xml-lsp-server/src/serverHelper.ts

```diff
@@ -0,0 +1,49 @@
+import {
```

Comment:
this repo does not use camel casing for file names.
use all lower characters with dashes instead.
`server-helper.ts`